Header header 
string Diagnostics
uint8 PodInfo
uint8 HMSCheck
uint8 OperationMode
uint8 SourceState
uint8 DestState